mitxgraders/helpers/mitmath/mathfuncs.py

```diff
@@ -255,9 +257,24 @@ def has_one_scalar_input(display_name):
 def has_one_square_input(display_name):
     return SpecifyDomain.make_decorator('square', display_name=display_name)
 
+def array_abs(obj):
+    """
+    Takes absolute value of numbers or vectors and suggests norm(...) instead
+    for matrix/tensors.
+
+    NOTE: The decision to limit abs(...) to scalars and vectors was motivated
+    by pedagogy not software.
+    """
+    if isinstance(obj, MathArray) and obj.ndim > 1:
```

Comment:
Very nitpicky here, but what happens to row/column matrices? (Do we even call these 'vectors'?) I'm likely to merge regardless, but just putting this out there as you think on #89.